docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,433 @@
+      ]
+    },
+    {
+      "metadata": {
+        "id": "6B95Hb6YVgPZ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Many of these imports/installations will not be needed in the near future.\n",
+        "!pip install tf-nightly-2.0-preview\n",
+        "import tensorflow as tf\n",
+        "import time\n",
+        "\n",
+        "# Load the TensorBoard notebook extension\n",
+        "%load_ext tensorboard.notebook\n",
+        "\n",
+        "# Clear any logs from previous runs\n",
+        "!rm -rf ./logs/  "
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "z5pr9vuHVgXY",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Use the [FashionMNIST](https://github.com/zalandoresearch/fashion-mnist) dataset as our example. "
+      ]
+    },
+    {
+      "metadata": {
+        "id": "j-DHsby18cot",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "mnist = tf.keras.datasets.mnist\n",
+        "\n",
+        "(x_train, y_train),(x_test, y_test) = mnist.load_data()\n",
+        "x_train, x_test = x_train / 255.0, x_test / 255.0\n",
+        "\n",
+        "def create_model():\n",
+        "  return tf.keras.models.Sequential([\n",
+        "    tf.keras.layers.Flatten(input_shape=(28, 28)),\n",
+        "    tf.keras.layers.Dense(512, activation=tf.nn.relu),\n",
+        "    tf.keras.layers.Dropout(0.2),\n",
+        "    tf.keras.layers.Dense(10, activation=tf.nn.softmax)\n",
+        "  ])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "XKUjdIoV87um",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Using TensorBoard with Keras Model.fit()"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "8CL_lxdn8-Sv",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "When training with Keras's [Model.fit()](https://www.tensorflow.org/api_docs/python/tf/keras/models/Model#fit), adding the TensorBoard callback will ensure the right logs are stored."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "WAQThq539CEJ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "model = create_model()\n",
+        "model.compile(optimizer='adam',\n",
+        "              loss='sparse_categorical_crossentropy',\n",
+        "              metrics=['accuracy'])\n",
+        "\n",
+        "tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=\"./logs/fit/{0}\".format(str(time.time())), histogram_freq=1)\n",
```

Comment:
It's worth explaining why we're appending the time to the logdir:

"We place the logs in a timestamped subdirectory to allow easy selection of different training runs."